raiden/tests/utils/eth_node.py

```diff
@@ -426,6 +430,23 @@ def _validate_jsonrpc_result(result):
             stack.enter_context(executor)
             executors.append(executor)
 
+            # The timeout_limit_teardown is necessary to prevent the build
```

Comment:
you refer to the timeout variable above as `timeout_limit_teardown` here (and also in transport.py HTTPExecutor), even though the variable is simply named `timeout`.

Is this timeout value supposed to be linked with the one the user sets and is configured through `conftest.py`? If so how?

EDIT: From reading the code again it seems as if you simply refer to the `timeout_limit_teardown` here. So you just want this timeout to be smaller than that?

If so why give the timeout value by poking the variable? Is the timeout given to the constructor the startup timeout and by poking the private vat you poke only the teardown one? Then can you perhaps show that by changing the name from `timeout` to `startup_timeout` ?

Reply:
> EDIT: From reading the code again it seems as if you simply refer to the timeout_limit_teardown here. So you just want this timeout to be smaller than that?

Yes, in total the runtime of the completely teardown must be smaller than `timeout_limit_teardown`, otherwise a timeout would happen. This is part of that teardown, since there are additional finalizers that have to run, this cannot use much of the teardown time, only a fraction of it.